bin/fin

```diff
@@ -1291,15 +1283,25 @@ show_help_image() {
 
 show_help_project() {
 	echo-green "Project management"
-	printh "fin project <command>" "" "yellow"
+	printh "fin project <command> [params]" "" "yellow"
 	echo
 	echo-green "Commands:"
-	printh "list (pl) [-a (--all)]" "List running Docksal projects (-a to show stopped as well)"
-	printh "create" "Create and install new Drupal, Wordpress or Magento project."
+	printh "start" "Start project services (shorthand: fin start)"
```

Comment:
"shortcut" maybe?
https://en.wikipedia.org/wiki/Shorthand

Reply:
Yeah it's not really a shortcut... It like and abbreviation. I think shorthand kinda describes what is happening here so unless you have hard feelings against this specific word I would stick with it.

Reply:
@david-hernandez do you have a recommendation as a native speaker? :)

Reply:
Neither is perfect, but shorthand is ok. I can't think of a better label at the moment, so its use is totally fine.

Reply:
The alternative might be to label it as alias.